docker-compose.java.yml

```diff
@@ -1,97 +1,6 @@
 version: '2.3'
 services:
-  postgres:
-    image: quay.io/azavea/postgis:2.3-postgres9.6-slim
-    volumes:
-      - ./data/:/tmp/data/
-    env_file: .env
-    expose:
-      - "5432"
-    healthcheck:
-      test: ["CMD", "pg_isready", "-U", "${POSTGRES_USER}"]
-      interval: 5s
-      timeout: 5s
-      retries: 3
-      start_period: 10s
-
-  sbt:
```

Comment:
We should be able to just use the `api-server` or `backsplash` service to invoke `./sbt`.

Reply:
One reason I am sure about is that `api-server` we couldn't because it didn't have the `gdal` bindings; if we use that image every where then we should be fine and can remove this I think.

The other reason was I think it didn't quite fit our development workflow since using one of those definitions would open ports that aren't necessary for compiling. A lot of times in development what I do now is the following:
 - In one shell I open `sbt` with `./scripts/console sbt` - this is where I switch to the project I'm working on and call `compile` or `assembly` if I want to test something in the server
 - In another shell I'll be starting the servers with `./scripts/server --java` 

If I am running `sbt` from one of the other task definition then the process will fail to start due to port conflicts. I think we could solve this by having the command not open service ports though.